packages/xo-server/src/xo-mixins/xen-servers.js

```diff
@@ -266,12 +266,14 @@ export default class {
       await xapi.connect()
 
       const xapisByPool = this._xapisByPool
+      const xapis = this._xapis
       const [{ $id: poolId }] = await xapi.getAllRecords('pool')
-      if (xapisByPool[poolId] !== undefined) {
+      if (get(() => xapis[xapisByPool[poolId]]) !== undefined) {
```

Comment:
Why is `get` necessary?

Reply:
IMHO `xapisByPool` is no longer correctly named.

Reply:
You're right is not necessary.